tutugodfrey/maintenance-tracker
# chore(test-signin): write test for signin

#### What does this PR do?
test signin endpoint

#### Description of Task to be completed?
/api/v1/users/signin is tested and working 

#### How should this be manually tested?
follow the instruction in project README on how to run test
ensure to set .env variable as specified in .env.example file

### What are the relevant pivotal tracker stories?
#157781719
